packages/Webkul/Core/src/Database/Migrations/2023_02_08_131145_add_column_translate_price_in_channels_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddColumnTranslatePriceInChannelsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('channels', function (Blueprint $table) {
+            $table->boolean('translate_price')->after('base_currency_id')->default(0);
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('channels', function (Blueprint $table) {
+            $table->dropColumn('translate_price');
+        });
+    }
```

Comment:
I think we can add it to the currency section because the currency is also linked to the channel and in this way, this will not get directly attached.